Guidelines.md

```diff
@@ -365,7 +365,7 @@ Services SHOULD return the following response headers, except where noted in the
 
 Response Header    | Required                                      | Description
 ------------------ | --------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-Date               | All responses                                 | The date the request was processed, in [RFC 3339][rfc-3339] format
+Date               | All responses                                 | Timestamp the response was processed, based on the server's clock, in [RFC 5322][rfc-5322-3-3] date and time format.  This is mandatory for response headers.  Coordinated Universal Time is the preferred time zone reference for this header,  designated as either `GMT` or `+0000`.  For example: `Wed, 24 Aug 2016 18:41:30 GMT`
 Content-Type       | All responses                                 | The content type
```

Comment:
Can we switch to normative language pls.  'Coordinated Universal Time SHOULD be used as the preferred time zone reference for this header".

Actually for the response, I think we should consider upping this to a MUST.  I can't see a reason for a service to return this in anything other than UTC.